web_embedding/element_embedding_demo/lib/main.dart

```diff
@@ -65,6 +65,7 @@ class _MyAppState extends State<MyApp> {
       title: 'Element embedding',
       theme: ThemeData(
         primarySwatch: Colors.blue,
```

Comment:
Judging at the screenshot, this color is not being applied. Is there any way of making the app use Colors.blue as its primary color, from which others are derived, so it matches better the design of the background?

Reply:
My understanding is `primarySwatch` is from Material 2. I believe it is better to replace with `colorSchemeSeed` or `colorScheme` for Material 3.

Context: https://api.flutter.dev/flutter/material/ThemeData/ThemeData.html